packages/client/src/features/tools/node-creation/node-creation-tool.ts

```diff
@@ -92,19 +92,21 @@ export class NodeCreationTool extends BaseCreationTool<TriggerNodeCreationAction
     }
 }
 
+export interface ContainerPositioningTool extends PositioningTool {
+    readonly containerManager: ContainerManager;
+}
```

Comment:
nit: I know we do not enforce this but I still would like to stay consistent within the file when it comes to empty lines. So could you please add an empty line after this declaration? If not, this PR will still be merged since I know it is just a preference ;-)